lib/ssl/test/ssl_ECC_SUITE.erl

```diff
@@ -405,3 +421,13 @@ mix_sign_rsa_peer(Config) ->
         ssl:filter_cipher_suites(ssl:cipher_suites(default, 'tlsv1.2'),
                                  [{key_exchange, fun(ecdhe_rsa) -> true; (_) -> false end}]),
     ssl_test_lib:basic_test(COpts, [{ciphers, ECDHE_RSA} | SOpts], Config).
+
+
+ecc_curves(TLSCurves) ->
+    CryptoCurves = crypto:supports(curves),
+    lists:foldr(fun(Curve, Curves) ->
+			case proplists:get_bool(Curve, CryptoCurves) of
```

Comment:
why using `proplists:get_bool/2` and not `lists:member/2` ? both seem to behave the same.
if CryptoCurves won't be a proplist then maybe `member` is easier to read and matching to input argument type.

Reply:
Well this was mainly done same way as in code in tls_v1. However I think that filtering for cryptolib support should be done in code instead of test so it will be lists:member/2 now.